app/views/play/cinematic/Command/AbstractCommand.js

```diff
@@ -0,0 +1,91 @@
+import * as Promise from 'bluebird'
```

Comment:
Tests for this file here: https://github.com/codecombat/codecombat/pull/5214/files#diff-69a90580d8295c30ed136fce03d2f1f2R2